assignment/scripts/describe.js

```diff
@@ -45,10 +43,10 @@ if (name === 'Mary') {
 */
 
 // 2. WRITE YOUR DESCRIPTION HERE
-// 
-// 
-// 
-//
+// We are creating a variable called code with the number 123.
+// We are creating a conditional statement where if code is strictly equalling 123 secret will show as a string of 'super', and multiply code by 2.
+// We are creating another conditional statement where if code is greater than 250 secret will output a string of 'duper'
+//console log of secret should show 'duper'
```

Comment:
Uh oh! 123 * 2 is only 246. The variable secret will be "super" after this code runs.